src/client/app/files/components/conference-group.js

```diff
@@ -67,7 +63,7 @@ export default ({ id, data }) => {
             ? <NonGroup><Emph>{`Other ${group.length} participants`}</Emph></NonGroup>
             : (
               <>
-                <div><Emph>Permalink: </Emph><a href={getPermalink(group)}>{getPermalink(group)}</a></div>
```

Comment:
destruct the group variable here, and pass only the sessionId to the getPermalink function, as it shouldn't know where the session id comes from.